packages/daimo-api/src/server/telemetry.ts

```diff
@@ -56,6 +59,20 @@ export class Telemetry {
         sampleRate: 1,
       });
     }
+
+    if (sentryDSN === "") {
+      console.log(`[TELEM] no SENTRY_DSN set, Sentry error reports disabled`);
+    } else {
+      Sentry.init({
+        dsn: sentryDSN,
+        integrations: [
+          new Sentry.Integrations.Postgres(),
+          nodeProfilingIntegration(),
+        ],
+        tracesSampleRate: 1.0,
+        profilesSampleRate: 1.0,
+      });
+    }
```

Comment:
I chose to set up Sentry in the telemetry file for two reasons:
- Keeping the `server.ts` file clean and
- Ensuring that it remains amongst the first things that happen when the server starts

That said, I can move it to another file if it is considered too distinct from the general telemetry functions.

Reply:
There are also a number of interesting things that can be achieved with the Sentry config. It might be interesting to experiment with a couple of them in another PR (mostly around profiling and instrumentation)